app/views/container_dashboard/show.html

```diff
@@ -77,4 +77,17 @@
             </div>
         </div>
     </div>
+    <div class="row row-tile-pf row-tile-pf-last">
+        <div class="col-md-6">
+            <div pf-card head-title="{{podsUtilizationHourlyConfig.headTitle}}">
+                <div pf-trends-chart config="podsUtilizationHourlyConfig" chart-data="hourlyPodsUtilization"
+                     ng-if="podsUtilizationLoadingDone" chart-height="chartHeight">
+                </div>
+                <div ng-if="!podsUtilizationLoadingDone" class="spinner spinner-lg loading"></div>
+            </div>
+        </div>
+        <div class="col-md-6">
+            <!-- place holder -->
```

Comment:
@abonas  Nothing.
do I need to remove only the remark, or the all div ? 
or may be change the remark words to state that it is just for placing the tile ?